jplag/src/test/java/de/jplag/special/TokenPrinterTest.java

```diff
@@ -43,7 +43,7 @@ void printJavaFiles() {
         });
     }
 
-    @Disabled
+    @Disabled("Not a meaningful test, used for designing the token set")
```

Comment:
good idea with the comment! simplifies the reading or increases the comprehensibility

Reply:
Yes! SonarCloud required me to do that, but it is actually very helpful.

Reply:
@dfuchss enabling SonarCloud already paid off! 😄